release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -108,6 +108,40 @@ For more information, see xref:../nodes/pods/nodes-pods-priority.html#non-preemp
 [id="ocp-4-7-monitoring"]
 === Monitoring
 
+[id="ocp-4-7-monitoring-prometheus-operator-alertmanagerconfig-crd-"]
+==== `AlertmanagerConfig` custom resource definition in Prometheus Operator not supported
+
+Modifying Alertmanager configurations by using the `AlertmanagerConfig` custom resource definition (CRD) in Prometheus Operator is not supported.
+
+For more information, see xref:../monitoring/configuring-the-monitoring-stack.html#support-considerations_configuring-the-monitoring-stack[Support considerations for monitoring].
+
+[id="ocp-4-7-monitoring-alerting-rule-changes"]
+==== Alerting rule changes
+
+{product-title} 4.7 includes the following alerting rule changes:
+
+* The `etcdBackendQuotaLowSpace` alert is added. The critical alert provides notification if the database size of an etcd cluster exceeds the defined quota on an etcd instance.
+* The `etcdHighFsyncDurations` alert is added. The critical alert provides notification if the 99th percentile fsync durations of an etcd cluster are too high.
+* The `etcdExcessiveDatabaseGrowth` alert is added. The warning alert provides notification if there is an observed surge in etcd writes that caused a 50% increase in database size on an etcd instance over a four-hour period.
+* The `NodeClockNotSynchronising` alert is updated ... // Need reason for update. I see that the `expr` changed to now include `and node_timex_maxerror_seconds >= 16`.
+* The `NodeNetworkReceiveErrs` alert is updated ... // Need reason for update. I see that the `expr` changed to `rate(node_network_receive_errs_total[2m]) / rate(node_network_receive_packets_total[2m]) > 0.01`.
+* The `NodeNetworkTransmitErrs` alert is updated ... // Need reason for update. I see that the `expr` changed to `rate(node_network_transmit_errs_total[2m]) / rate(node_network_transmit_packets_total[2m]) > 0.01`.
+*
+
+[NOTE]
+====
+Red Hat does not guarantee backward compatibility for metrics, recording rules, or alerting rules.
+====
+
+[id="ocp-4-6-monitoring-namespace-by-pod-and-pod-total-Grafana-dashboards"]
+==== Namespace By Pod and Pod Total Grafana dashboards
+
+The Namespace By Pod and Pod Total networking Grafana dashboards are enabled. // What are these dashboards used for?
+
+[id="ocp-4-6-monitoring-prometheus-config-reloader-memory-limits"]
+==== `prometheus-config-reloader` memory limit removed for monitoring user defined projects
+
+The memory limit was removed on the Prometheus Operator `prometheus-config-reloader` container in the `openshift-user-workload-monitoring` namespace for monitoring user defined projects. Memory limits are not set on containers. // More info needed?
```

Comment:
The reason behind this change is to avoid the `prometheus-config-reloader` container being killed because it uses more memory than the defined limit.